wp-dependency-installer.php

```diff
@@ -83,28 +97,29 @@ public function load_hooks() {
 		/**
 		 * Register wp-dependencies.json
 		 *
-		 * @param string $plugin_path Path to plugin or theme calling the framework.
+		 * @param string $plugin_path (optional) Path to plugin or theme calling the framework.
 		 */
-		public function run( $plugin_path ) {
+		public function run( $plugin_path = false ) {
+			$plugin_path = empty( $plugin_path ) ? $this->plugin_path : $plugin_path; // TODO: deprecate parameter in future releases?
```

Comment:
Should this be `$plugin_path = ! $plugin_path ? $this->plugin_path : $plugin_path;`